js/flightlog_fields_presenter.js

```diff
@@ -511,7 +511,7 @@ function FlightLogFieldPresenter() {
             case 'motor[5]':            
             case 'motor[6]':            
             case 'motor[7]':            
-                return Math.round(flightLog.rcMotorRawToPct(value)) + " %";
+                return `${Math.round(flightLog.rcMotorRawToPct(value))} %`;
```

Comment:
With a float we get more precision?

PWM | Dshot | MotorPct
-- | -- | --
1000 | 0 | -2.40120
1001 | 48 | 0.00000
1002 | 50 | 0.10010
1003 | 52 | 0.20020
1004 | 54 | 0.30030
1005 | 56 | 0.40040
1006 | 58 | 0.50050
1007 | 60 | 0.60060
1008 | 62 | 0.70070
1009 | 64 | 0.80080
1010 | 66 | 0.90090
1011 | 68 | 1.00100
1012 | 70 | 1.10110
1013 | 72 | 1.20120
1014 | 74 | 1.30130
1015 | 76 | 1.40140

Reply:
Yes, I commented it somewhere to add one decimal to the percentage (more than one is too much).

Another solution is to let the percentage, and add the "real" motor value:

For example: `23 % (46)` or the inverse `46 (23 %)`.

Reply:
In true DShot values, we have 1000 steps?  So perhaps one decimal point on the percentage?